src/resources/Subscriptions.ts

```diff
@@ -256,18 +258,25 @@ export class ScheduledPayments extends CRUDResource {
     }
 }
 
+export const hasImmediateCharge = (subscription: SubscriptionItem) => {
```

Comment:
doesn't seem like you need to export this 🤔

Reply:
Right. Over-engineering. I thought it would be useful if we want to do some custom implementation still but you are right, this is not needed, Let me remove the export 🙂